tests/rules/test_brew_cask_reinstall.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+from thefuck.types import Command
+from thefuck.rules.brew_cask_reinstall import get_new_command, match
+
+
+output = ("Warning: Cask 'thefuck' is already installed.\n\nTo "
+          "re-install thefuck, run\n  `brew cask reinstall thefuck`")
```

Comment:
The actual output is:
```
Warning: Cask 'thefuck' is already installed.

To re-install docker, run:
  brew cask reinstall thefuck
```

```suggestion
          "re-install thefuck, run:\n  brew cask reinstall thefuck")
```